Services/SnippetsReader.php

```diff
@@ -0,0 +1,122 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * Created by solutionDrive GmbH
+ *
+ * @copyright solutionDrive GmbH
+ */
+
+namespace sdDeploymentHelperShopware\Services;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Inspired by \Shopware\Components\Snippet\DatabaseHandler
+ */
+class SnippetsReader implements SnippetsReaderInterface
+{
+    /** @var string */
+    private $kernelRootDir;
+
+    /** @var OutputInterface */
+    private $output;
+
+    public function __construct(
+        string $kernelRootDir
+    ) {
+        $this->kernelRootDir = $kernelRootDir;
+    }
+
+    /**
+     * @param OutputInterface $output
+     */
+    public function setOutput(OutputInterface $output)
+    {
+        $this->output = $output;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function readSnippets(
+        string $snippetsDir = null
+    ): array {
+        $snippets = [];
+        $snippetsDir = $this->getSnippetsDir($snippetsDir);
+        if (!file_exists($snippetsDir)) {
+            $this->checkSnippetsDir($snippetsDir);
+
+            return [];
+        }
+
+        $inputAdapter = new \Enlight_Config_Adapter_File([
+            'configDir' => $snippetsDir,
+        ]);
+
+        $finder = new Finder();
+        $finder->files()->in($snippetsDir);
+
+        foreach ($finder as $file) {
+            $filePath = $file->getRelativePathname();
+            if ('.ini' === $file->getExtension()) {
```

Comment:
The dot must be removed (http://php.net/manual/de/splfileinfo.getextension.php)

Reply:
done!